contrib/mergeBenchmarkSets.py

```diff
@@ -82,102 +111,97 @@ def getWitnessResult(witness, verification_result):
     return "result invalid (" + status_from_verification + ")", result.CATEGORY_ERROR
 
 
-def main(argv=None):
-
-    if argv is None:
-        argv = sys.argv
+def get_validation_result(
+    run, witness_sets, status_from_verification, category_from_verification
+):
+    status_wit, category_wit = None, None
+    name = run.get("name")
+    for witnessSet in witness_sets:
+        witness = witnessSet.get(name)
+        if not witness:
+            continue
```

Comment:
This case is not currently covered by any of the unit tests.

Reply:
This is covered by `test_getValidationResult_multiple_witnesses`. It also doesn't need changing because it makes perfect sense to have some witnesses not in every witness set, e.g., if a tool only validated correctness witnesses.